From 7919c2fd9e07276403b9a4d9ae52305e0d70f923 Mon Sep 17 00:00:00 2001
From: Johan Hovold <jhovold@gmail.com>
Date: Wed, 17 Mar 2010 23:00:41 +0100
Subject: USB: pl2303: increase bulk-in buffer size to 256 byte
Patch-mainline: v2.6.35
Git-commit: 7919c2fd9e07276403b9a4d9ae52305e0d70f923

The pl2303 requires a bulk-in buffer larger than endpoint size to keep
up at high baudrates without loosing data.

Signed-off-by: Johan Hovold <jhovold@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/pl2303.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/serial/pl2303.c
+++ b/drivers/usb/serial/pl2303.c
@@ -1191,6 +1191,7 @@ static struct usb_serial_driver pl2303_d
 	.id_table =		id_table,
 	.usb_driver = 		&pl2303_driver,
 	.num_ports =		1,
+	.bulk_in_size =		256,
 	.bulk_out_size =	256,
 	.open =			pl2303_open,
 	.close =		pl2303_close,
